source/hp/dof_handler.cc

```diff
@@ -1671,6 +1672,19 @@ namespace hp
             ExcMessage ("The given triangulation is parallel distributed but "
                         "this class does not currently support this."));
 
+    // decide whether we need a sequential or a parallel distributed policy
```

Comment:
No need -- I'll get to that within the next few days hopefully. This is the next place I'll have to fix if I want `hp::DoFHandler` to make in parallel. Not a place I can possibly forget because I'll need a different policy in that case.